### ISSUE TITLE
[ERROR] plugin.child.apps: plugin process exited: while uploading app manifest

### ISSUE THREAD

**Issue by R-Inbarasu** [stripe/stripe-apps#780]
I'm facing issue while trying to upload the app.

MT202102-0153:backend inbarasu$ stripe apps upload --log-level debug
[Tue, 20 Dec 2022 11:30:28 IST] DEBUG config.Config.InitConfig: Using profiles file path=/Users/inbarasu/.config/stripe/config.toml
[Tue, 20 Dec 2022 11:30:28 IST] DEBUG config.Config.GetProfilesFolder: Using profiles file path=/Users/inbarasu/.config/stripe
[Tue, 20 Dec 2022 11:30:28 IST] DEBUG cmd.pluginCmd.runPluginCmd: Running plugin...
[Tue, 20 Dec 2022 11:30:28 IST] DEBUG config.Config.GetProfilesFolder: Using profiles file path=/Users/inbarasu/.config/stripe
[Tue, 20 Dec 2022 11:30:28 IST] DEBUG config.Config.GetProfilesFolder: Using profiles file path=/Users/inbarasu/.config/stripe
⬆ You are about to upload your Stripe App Gist for Stripe version "1.0.0" to Mallow-tech.
Would you like to proceed: Y
2022-12-20T11:30:38.756+0530 [ERROR] plugin.child.apps: plugin process exited: path=/Users/inbarasu/.config/stripe/plugins/apps/1.5.1/stripe-cli-apps pid=27305 error="exit status 2"
[Tue, 20 Dec 2022 11:30:38 IST] DEBUG Tearing down plugin before exit
[Tue, 20 Dec 2022 11:30:38 IST] DEBUG pluginTemplateCmd.runPluginCmd: Plugin command 'apps' exited with error: unexpected EOF

stripe version 1.13.6

Issue already reported here(https://github.com/stripe/stripe-apps/issues/770). But it was closed without any explanation on the issue.

**Comment by msindwan-stripe** [comment]
Hi @R-Inbarasu, can you confirm if you're experiencing the same issue with a different app bundle when attempting an upload?

**Comment by R-Inbarasu** [comment]
No, I created a new bundle and uploaded it after changing the manifest file to match my requirement and it uploaded without any issue.

**Comment by msindwan-stripe** [comment]
@R-Inbarasu glad to see that you were able to upload the app! If you're comfortable, please feel free to share the initial manifest file and or source code for us to debug further. Otherwise, please let us know if you run into any other issues
